React/Modules/RCTAsyncLocalStorage.m

```diff
@@ -170,6 +190,14 @@ - (dispatch_queue_t)methodQueue
   return RCTGetMethodQueue();
 }
 
+- (NSDictionary *)constantsToExport
```

Comment:
`constantsToExport` prevents the module to be lazy loaded, so when it is possible (like this case), we have to move this to JavaScript side.